docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+
+```
+@my_data.h2o_temperature{location, state, surface_degrees, time}
+```
+
+Would return the last value of surface_degrees in each series:
+
+| location | state | surface_degrees | time |
+| --- | --- | --- | --- |
+| coyote_creek | CA | 50.2 | 1600756160 |
+| puget_sound | WA | 54.7 | 1600756160 |
+| santa_monica | CA | 63.6 | 1600756160 |
+
+
+In Flux, a tag key and a field key are both called columns. This means that tag keys and field keys must
+be unique within a measurement. Here's an example that explicitly selects all the columns we know about:
+
+```
+@my_data.h2o_temperature{location, state, surface_degrees, bottom_degrees, time}
+``` 
+
+We can also select all columns by leaving the column list empty:
+
+```
+@my_data.h2o_temperature{}
+``` 
+
+Or you can leave out the curly braces:
+
+```
+@my_data.h2o_temperature
+```
+
+If your **Column Filter Predicate** selects only a tag, you'll get the same number of records as you have series for 
+that tag since the last value for each series is what gets defined if you're not filtering by time range or other 
+criteria.
+
+```
+@my_data.h2o_temperature{state}
+```
+
+Returns these results:
+
+| state |
+| --- |
+| CA |
+| CA |
+| WA |
+
+#### Filtering on Criteria
+You can limit the results returned in the query by specifying filtering criteria in the *Column Filter Predicate* block. 
+Here's an example that filters based on the location:
+
+```
+@my_data.h2o_temperature{location, surface_degrees, time, state == "CA"}
+```
+
+| location | state | surface_degrees | time |
+| --- | --- | --- | --- |
+| coyote_creek | CA | 50.2 | 1600756160 |
+| santa_monica | CA | 63.6 | 1600756160 |
+
+Note that `state` is returned in the result because it was listed in the filter criteria. You can have more complex 
+logic to match criteria like:
+
+```
+@my_data.h2o_temperature{location, surface_degrees, time, location == "coyote_creek" OR location == "puget_sound"}
+```
+
+See that while the column names are separated by commas, the filter criteria is separated by the different matching 
+clauses.
+
+
+#### Matching Critera
+There are many operators you can use to filter down data. Here is the list of operators:
+* `==` (equal to)
+* `!=` (not equal)
+* `=~` (regex match)
+* `!~` (not a regex match)
+* `>` (greater than)
+* `>=` (greater than or equal to)
+* `<` (less than)
+* `<=` (less than or equal to)
+* `in`
+
+The regex and in operators are probably the trickiest ones to work with, so here are a few examples:
+
+```
+// this is a comment because it has // before it
+
+// status is a tag and this will match any stats between 200 and 500
+@my_data.http_requests{status =~ /[2-5][0-9][0-9]/}
+
+// will match states in the array
+@my_data.h2o_temperature{state in ["CA", "WA"]}
+```
+
+### Aggregate, Select, Transform and Group Data
+Flux has many built in functions to work with time series data. **Aggregate** functions refer to functions that summarize 
```

Comment:
I wonder if we can take a page from Telegraf here: https://github.com/influxdata/telegraf/blob/master/docs/AGGREGATORS_AND_PROCESSORS.md#aggregator

Aggregate function "emit new metrics for some period of time"